modules/ztp-configuring-pgt-compliance-eval-timeouts.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/ztp-far-edge/ztp-deploying-far-edge-sites.adoc
+
+:_content-type: PROCEDURE
+[id="ztp-configuring-pgt-compliance-eval-timeouts_{context}"]
+= Configuring policy compliance evaluation timeouts for PolicyGenTemplate CRs
+
+Use {rh-rhacm-first} installed on a hub cluster to monitor and report on whether your managed clusters are compliant with applied policies. {rh-rhacm} uses policy templates to apply predefined policy controllers and policies. Policy controllers are Kubernetes custom resource definition (CRD) instances.
+
+You can override the default policy evaluation intervals with `PolicyGenTemplate` custom resources (CRs). You configure duration settings that define how long a `ConfigurationPolicy` CR can be in a state of policy compliance or non-compliance before {rh-rhacm} re-evaluates the applied cluster policies.
+
+The zero touch provisioning (ZTP) policy generator generates `ConfigurationPolicy` CR policies with pre-defined policy evaluation intervals. The default value for the `noncompliant` state is 10 seconds. The default value for the `compliant` state is 10 minutes. To disable the evaluation interval, set the value to `never`.
+
+.Prerequisites
+
+* You have installed the OpenShift CLI (`oc`).
+
+* You have logged in to the hub cluster as a user with `cluster-admin` privileges.
+
+* You have created a Git repository where you manage your custom site configuration data.
+
+.Procedure
+
+. To configure the evaluation interval for all policies in a `PolicyGenTemplate` CR, add `evaluationInterval` to the `spec` field, setting appropriate `compliant` and `noncompliant` values, for example:
++
+[source,yaml]
+----
+spec:
+  evaluationInterval:
+    compliant: 30m
+    noncompliant: 20s
+----
+
+. To configure the evaluation interval for the `spec.sourceFiles` object in a `PolicyGenTemplate` CR, add `evaluationInterval` to the `sourceFiles` field, for example:
++
+[source,yaml]
+----
+spec:
+  sourceFiles:
+   - fileName: SriovSubscription.yaml
+     policyName: "sriov-sub-policy"
+     evaluationInterval:
+       compliant: never
+       noncompliant: 10s
+----
+
+. Commit the `PolicyGenTemplate` CRs files in the Git repository and push your changes.
+
+.Verification
```

Comment:
I believe your preview might be out of date. The Verification steps are not in the published output. Please update, so I can verify that it looks OK.

Reply:
/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done